.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -65,7 +65,7 @@ const completedContributorPlusChecklist = `- [x] I have verified the author chec
     - [x] Android / Chrome
     - [x] MacOS / Chrome
     - [x] MacOS / Desktop
-- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
+- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
```

Comment:
oh wait no don't you need to change this too https://github.com/Expensify/App/blame/main/.github/PULL_REQUEST_TEMPLATE.md#L53?

Reply:
That is a PR author checklist and we are changing the C+ checklist